detox/test/e2e/28.webview.test.js

```diff
@@ -0,0 +1,199 @@
+const MOCK_TEXT = 'Mock Text';
+
+describe(':android: WebView', () => {
+
+  beforeEach(async () => {
+    await device.reloadReactNative();
+    //detox
+    await element(by.text('WebView')).tap();
+  });
+
+  async function getWebView(matcher) {
+    return web.getWebView(matcher !== undefined ? matcher : by.id('webview_1'));
+  }
+
+  describe('Expectations',() => {
+    it('expect element to exists', async () => {
+      //detox web
+      webview = await getWebView();
+
+      await web.expect(webview.element(web.by.id('testingPar'))).toExists();
```

Comment:
did you try testing such a thing before going into implementation?

Reply:
@OrenZak FYI @noomorph is searching for alternative solutions.

Reply:
@noomorph thinking out loud: How about replacing `getWebView()` altogether, with our own `with`-like variation, such as `withWebView(lambda)` (name is just a suggestion). That would yield code that would like this, more or less:
```js
await withWebView(() => {
   await expect(element(by.id('testingPar')).toExist();
   
   const editable = await element(by.className('public-DraftEditor-content'));
   await editable.scrollToView();
});
```
inside the scope of `withWebView()` (and inside it only...), we would internally flip a switch such that the global `element` and `by` would refer to the _web_-ish ones, instead of the defaults.
A bit tricky to implement but keeps the API much cleaner...

🤔

Reply:
or maybe we could use `with(webview)` internally, scoping it to the call of the user's lambda - thus enabling the initial suggestion without asking the users from implementing discouraged practices in their code...